src/main/java/gregtech/integration/jei/multiblock/MultiblockInfoPage.java

```diff
@@ -19,10 +19,12 @@ public float getDefaultZoom() {
 
     public Map<MultiblockAbility, List<String>> getAbilityTooltipMap() {
         Map<MultiblockAbility, List<String>> abilityTooltipMap = new HashMap<>();
-        abilityTooltipMap.put(MultiblockAbility.EXPORT_ITEMS, Collections.singletonList("gregtech.multiblock.preview.any_hatch"));
-        abilityTooltipMap.put(MultiblockAbility.IMPORT_ITEMS, Collections.singletonList("gregtech.multiblock.preview.any_hatch"));
-        abilityTooltipMap.put(MultiblockAbility.EXPORT_FLUIDS, Collections.singletonList("gregtech.multiblock.preview.any_hatch"));
-        abilityTooltipMap.put(MultiblockAbility.IMPORT_FLUIDS, Collections.singletonList("gregtech.multiblock.preview.any_hatch"));
+        List<String> tooltipList = new ArrayList<>();
+        tooltipList.add("gregtech.multiblock.preview.any_hatch");
+        abilityTooltipMap.put(MultiblockAbility.EXPORT_ITEMS, tooltipList);
+        abilityTooltipMap.put(MultiblockAbility.IMPORT_ITEMS, tooltipList);
+        abilityTooltipMap.put(MultiblockAbility.EXPORT_FLUIDS, tooltipList);
```

Comment:
You are sharing the same list between the abilities?

So my previous attempt to add a tooltip to the export items would also be added to the others.

How about implementing it this way?

````
// Cached version
private Map<MultiblockAbility, List<String>> abilityTooltips;

public Map<MultiblockAbility, List<String>> getAbilityTooltipMap() {
    // Generate on first use
    if (this.abilityTooltips == null) {
        this.abilityTooltips = Maps.newHashMap();
        generateAbilityTooltips();
    }
    return this.abilityToolTips;
}

// Helper method
public void addAbilityTooltip(MulitblockAbility ability, String tooltip) {
    List<String> tooltips = this.abilityToolTips.putIfAbsent(ability, Lists.newArrayList());
    tooltips.add(tooltip);
}

// The default implementation
public void generateAbilityTooltips() {
    addAbilityToolTip(MultiblockAbility.EXPORT_ITEMS, "gregtech.multiblock.preview.any_hatch");
    addAbilityToolTip(MultiblockAbility.IMPOTY_ITEMS, "gregtech.multiblock.preview.any_hatch");
    addAbilityToolTip(MultiblockAbility.EXPORT_FLUIDS, "gregtech.multiblock.preview.any_hatch");
    addAbilityToolTip(MultiblockAbility.IMPORT_FLUIDS, "gregtech.multiblock.preview.any_hatch");
}
````

This is easier to use (harder to get wrong) and would make my subclassing example less verbose
````
@Override
public void generateAbilityTooltips() {
    super.generateAbilityTooltips();
    addAbilityToolTip(MultiblockAbility.EXPORT_ITEMS, "my.tooltip");
}
````

It also adds caching on first use.

Reply:
There is still a problem with the way you are proposing warjot : you can't override existing tooltip without rewriting the main par of  `generateAbilityTooltips()`
I think a `removeTooltip(MulitblockAbility ability, String tooltip)` method might be useful.

Reply:
I don't think that remove is very useful? 
You can remove tooltips you know about, but later additions to the default processing that you don't know about  at development time might still be wrong.

If you really are overridding behaviour such that default tooltips are wrong, you probably shouldn't be calling super.

Reply:
well i think the multiblock which require a minimal or very specific tier for some hatches may want to use super for other hatches or in case the default tooltips added changes. I think the steam multiblock from gregicality could be an exemple.